pmlc/.clang-format

```diff
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-AlwaysBreakTemplateDeclarations: Yes
+AlwaysBreakTemplateDeclarations: true
```

Comment:
Which version of `clang-format` are you using? I believe we require version 9.

Reply:
ah, apparently 6.something, I'll update, thanks